pkg/argocd/repo-server.go

```diff
@@ -80,29 +80,39 @@ func createRepoServerDeployment(clientset *kubernetes.Clientset, namespace, argo
 								},
 							},
 						},
+						{
+							Name: "gpg-keys",
+							VolumeSource: corev1.VolumeSource{
+								EmptyDir: &corev1.EmptyDirVolumeSource{},
+							},
+						},
 					},
 					Containers: []corev1.Container{
-						corev1.Container{
+						{
 							Name:  "argocd-repo-server",
 							Image: argoImage,
 							Command: []string{
 								"uid_entrypoint.sh",
 								"argocd-repo-server",
 							},
 							Ports: []corev1.ContainerPort{
-								corev1.ContainerPort{
+								{
 									ContainerPort: 8081,
 								},
 							},
 							VolumeMounts: []corev1.VolumeMount{
-								corev1.VolumeMount{
+								{
 									Name:      "ssh-known-hosts",
 									MountPath: "/app/config/ssh",
 								},
-								corev1.VolumeMount{
+								{
 									Name:      "tls-certs",
 									MountPath: "/app/config/tls",
 								},
+								{
+									Name:      "gpg-keys",
+									MountPath: "/app/config/gpg",
```

Comment:
```suggestion
									MountPath: "/app/config/gpg/keys",
```

See https://github.com/argoproj/argo-cd/blob/master/manifests/base/repo-server/argocd-repo-server-deployment.yaml#L44

Reply:
I just figured that out by trial-and-error :) I have tested it against my test cluster and it works